### PR TITLE
Updated the MQTT endpoint variable name as updated in the latest branch.

### PR DESCRIPTION
**Pull request by HarnishaA**
Updated the MQTT endpoint variable name as updated in the latest branch.

Description
-----------
Updated the MQTT endpoint variable name as updated in the latest branch in aws_credentials.h file. 

Checklist:
----------

- [x] I have tested my changes. No regression in existing tests.
- [x] My code is Linted.

### CODE REVIEW

File: tools/ota_e2e_tests/aws_ota_test/aws_ota_project.py [hunk 1]
```diff
@@ -219,7 +222,7 @@ def setClientCredentialsForAwsIotEndpoint(self, awsIotEndpoint, awsIotEndpointPo
         """
         self.__setIdentifierInFile(
             {
-                'static const char clientcredentialMQTT_BROKER_ENDPOINT[] =': '\"' + awsIotEndpoint + '\";',
+                '#define clientcredentialMQTT_BROKER_ENDPOINT ': '\"' + awsIotEndpoint + '\"',
```
Comment (sarenameas) [inline tools/ota_e2e_tests/aws_ota_test/aws_ota_project.py]: __setIdentifierInFile() already inserts a space after the key, although I think it does not hurt to have one here.

File: tools/ota_e2e_tests/aws_ota_test/aws_ota_project.py [hunk 2]
```diff
@@ -134,6 +136,7 @@ def buildProject(self):
             print('====> Command run completed with the return code: ', proc.returncode)
             returnCodes.append(proc.returncode)
 
+        sleep(5)
```
Comment (sarenameas) [inline tools/ota_e2e_tests/aws_ota_test/aws_ota_project.py]: What is this sleep for? Is to give the process time to generate binaries after returning?

Comment (HarnishaA) [reply tools/ota_e2e_tests/aws_ota_test/aws_ota_project.py]: Yeah .. It is causing the test to fail saying could not find output binary even after its actually present. Just added sleep to test. If this works i'll add proper handling for this scenario.

File: tools/ota_e2e_tests/aws_ota_test/aws_ota_project.py (hunk 1 above)
Comment (HarnishaA) [reply tools/ota_e2e_tests/aws_ota_test/aws_ota_project.py]: Will remove it in the next commit to keep the consistency :-)

File: tools/ota_e2e_tests/aws_ota_test/aws_ota_project.py (hunk 2 above)
Comment (sarenameas) [reply tools/ota_e2e_tests/aws_ota_test/aws_ota_project.py]: Maybe we can add a comment if you plan to keep it?

Comment (HarnishaA) [reply tools/ota_e2e_tests/aws_ota_test/aws_ota_project.py]: Added the comment in the below commit :-)

**Pull request merged**